### PR TITLE
Add a config option to control the default widget container height

### PR DESCRIPTION
**Pull request by dbkr**
See https://github.com/element-hq/element-web/pull/27922 for docs

## Checklist

-   [ ] Tests written for new code (and old code if feasible).
-   [ ] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
-   [ ] Linter and other CI checks pass.
-   [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md)).

### CODE REVIEW

File: src/SlidingSyncManager.ts [hunk 1]
```diff
@@ -383,7 +383,7 @@ export class SlidingSyncManager {
     public async nativeSlidingSyncSupport(client: MatrixClient): Promise<boolean> {
         // Per https://github.com/matrix-org/matrix-spec-proposals/pull/3575/files#r1589542561
         // `client` can be undefined/null in tests for some reason.
-        const support = await client?.doesServerSupportUnstableFeature("org.matrix.msc3575");
+        const support = await client?.doesServerSupportUnstableFeature("org.matrix.simplified_msc3575");
```
Comment (t3chguy) [inline src/SlidingSyncManager.ts:386]: this seems unrelated to this PR

**Review by MidhunSureshR — APPROVED** [review]
